-------------------------------------------------------------------------------
Test set: org.simplity.service.test.ActionsWithDBIT
-------------------------------------------------------------------------------
Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 3.03 sec <<< FAILURE! - in org.simplity.service.test.ActionsWithDBIT
setValueTest(org.simplity.service.test.ActionsWithDBIT)  Time elapsed: 1.909 sec  <<< ERROR!
org.simplity.kernel.ApplicationError: 
java.net.ConnectException: Connection refused: connect
java.net.ConnectException: Connection refused: connect
Connection refused: connect

	at org.simplity.service.test.ActionsWithDBIT.setUp(ActionsWithDBIT.java:94)

LogicTest(org.simplity.service.test.ActionsWithDBIT)  Time elapsed: 1.021 sec  <<< ERROR!
org.simplity.kernel.ApplicationError: 
java.net.ConnectException: Connection refused: connect
java.net.ConnectException: Connection refused: connect
Connection refused: connect

	at org.simplity.service.test.ActionsWithDBIT.setUp(ActionsWithDBIT.java:94)

